tests/event_test.py

```diff
@@ -103,3 +103,46 @@ def test_wait_timeout_exceed():
         td = eventlet.hubs.get_hub().clock() - t1
         assert not result
         assert td >= delay
+
+
+def test_no_mem_leaks():
```

Comment:
This is not a test that tests the `Event` class, so most likely the code needs to be moved to another file. `greenthread_test.py` seems appropriate after a quick glance if it's actually `GreenThread` that's gonna be changed.

Reply:
it is seen from using greenthread,
while the issue is generated in the Event instance
instance of Event is keeping in it's self._exc the frames trace including a greenthead instance/s.
It could have been anything else using Event

Reply:
That's fair, I misread the code.